pkgs/tools/package-management/akku/overrides.nix

```diff
@@ -0,0 +1,120 @@
+{ stdenv, lib, akku, akkuPackages, curl, git, substituteAll }:
+let
+  joinOverrides =
+    overrides: pkg: old:
+    lib.attrsets.mergeAttrsList (map (o: o pkg old) overrides);
+  addToBuildInputs =
+    extras: pkg: old:
+    { propagatedBuildInputs = old.propagatedBuildInputs ++ extras; };
+  broken = lib.addMetaAttrs { broken = true; };
+  skipTests = pkg: old: { doCheck = false; };
+  # debugging
+  showLibs = pkg: old: { preCheck = "echo $CHEZSCHEMELIBDIRS"; };
+  runTests = pkgs: old: { doCheck = true; };
+in
+{
+  chez-srfi = pkg: old: {
+    preCheck = ''
+      SKIP='
+      multi-dimensional-arrays.sps
+      time.sps
+      tables-test.ikarus.sps
+      lazy.sps
+      '
+    '';
+    # doCheck = false; # for development bc rebuilds take forever
```

Comment:
```suggestion
    # Exception in get-environment-variables: currently unsupoorted on ~s with irritant tarm64osx
    doCheck = !(stdenv.isDarwin && stdenv.isAarch64);
```

Reply:
Oh, there are other issues on aarch64-darwin, I guess akku should be marked as broken there. x86_64-darwin is fine.